eng/common/testproxy/publish-proxy-logs.yml

```diff
@@ -1,16 +1,24 @@
 parameters:
   rootFolder: '$(Build.SourcesDirectory)'
+  OneES: false
```

Comment:
Is there any global parameters or variables set in the 1es case we can use? If not adding a new one should be fine.